src/LiveSplit.Core/Server/CommandServer.cs

```diff
@@ -291,6 +294,94 @@ private void ProcessMessage(string message, IConnection clientConnection)
+                {
+                    region =  State.Run.Metadata.Region.Abbreviation;
+                }
+                //Platform
+                string platform = null;
+                bool doSimplePlatform = !State.Run.Metadata.GameAvailable;
+                if (!string.IsNullOrEmpty(State.Run.Metadata.PlatformName) && (doSimplePlatform || (State.Run.Metadata.Game != null && State.Run.Metadata.Game.Platforms.Count > 1)))
+                {
+                    if (State.Run.Metadata.UsesEmulator)
+                    {
+                        platform = $"{State.Run.Metadata.PlatformName} Emulator";
+                    }
+                    else
+                    {
+                        platform = State.Run.Metadata.PlatformName;
+                    }
+                }
+                else if (State.Run.Metadata.UsesEmulator)
+                {
+                    platform = "Emulator";
+                }
+                //Variables
+                Dictionary<string, string> varDict = new Dictionary<string, string>();
+                IEnumerable<string> variables = State.Run.Metadata.VariableValueNames.Keys;
+                if (State.Run.Metadata.GameAvailable && State.Run.Metadata.Game != null)
+                {
+                    string categoryId = null;
+                    if (State.Run.Metadata.CategoryAvailable && State.Run.Metadata.Category != null)
+                    {
+                        categoryId = State.Run.Metadata.Category.ID;
+                    }
+                    
+                    variables = State.Run.Metadata.Game.FullGameVariables.Where(x => x.CategoryID == null || x.CategoryID == categoryId).Select(x => x.Name);
+                }
+
+                foreach (string variable in variables)
+                {
+                    if (State.Run.Metadata.VariableValueNames.ContainsKey(variable))
+                    {
+                        string name = variable.TrimEnd('?');
+                        string variableValue = State.Run.Metadata.VariableValueNames[variable];
+                        string valueLower = variableValue.ToLowerInvariant();
+                        if (valueLower == "yes")
+                        {
+                            varDict.Add(variable, name);
+                        }
+                        else if (valueLower == "no")
+                        {
+                            varDict.Add(variable, $"No {name}");
+                        }
+                        else
+                        {
+                            varDict.Add(variable, variableValue);
+                        }
+                    }
+                }
+                //Combine
+                CategoryMetaData metadata = new CategoryMetaData(region, platform, varDict.Count() > 0 ? varDict : null);
```

Comment:
```suggestion
                CategoryMetaData metadata = new(region, platform, varDict.Count() > 0 ? varDict : null);
```

Your editor should have suggested this change to you. Please don't ignore style-related diagnostics.

Reply:
I did not get something for this and I'm on the VS setup from the README.

Reply:
On a completely fresh install of Visual Studio and a fresh clone of your branch, I do get these diagnostics:
![image](https://github.com/user-attachments/assets/a6f5889c-146e-4720-bd12-a44d44967c16)
![image](https://github.com/user-attachments/assets/81e8e580-dacf-464a-80b7-ff9dd22e85d1)

Reply:
I've should clarify that particular change wasn't suggested, but I did get some that were likely worse. Sorry about that!